src/plugins/unified_search/public/dataview_picker/dataview_list.tsx

```diff
@@ -33,6 +67,54 @@ export function DataViewsList({
   selectableProps,
   searchListInputId,
 }: DataViewsListProps) {
+  const { euiTheme } = useEuiTheme();
+  const popoverStyle = euiTheme.base * 13;
+  const kibana = useKibana<IUnifiedSearchPluginServices>();
+  const { storage } = kibana.services;
+
+  const [isSortingPopoverOpen, setIsSortingPopoverOpen] = useState(false);
+
+  const [sortedDataViewsList, setSortedDataViewsList] = useState(dataViewsList);
+
+  const [sortByOptions, setSortByOptions] = useState<EuiSelectableOption[]>(() => {
+    return getCompatibleSortingTypes().map((key) => {
+      return {
+        data: { sortBy: key },
+        checked: key === DEFAULT_SORT.by ? 'on' : undefined,
+        label: OptionsListStrings.editorAndPopover.sortBy[key].getSortByLabel(),
+      };
+    });
+  });
+
+  const [sortOrderOptions, setSortOrderOptions] = useState<EuiSelectableOption[]>(() => {
+    return getCompatibleSortingTypesByOrder().map((key) => {
+      return {
```

Comment:
is it possible to extract 

```
return {
        data: { sortBy: key },
        checked: key === DEFAULT_SORT.by ? 'on' : undefined,
        label: OptionsListStrings.editorAndPopover.sortBy[key].getSortByLabel(),
      };
```

into separate method like e.g. toSelectableOptions